checkita-core/src/main/scala/ru/raiffeisen/checkita/storage/Models.scala

```diff
@@ -104,6 +115,15 @@ object Models {
                                    executionDate: Timestamp) extends CheckResult {
     val entityType: String = "loadCheckResult"
   }
+
+  final case class ResultJobConfig(
```

Comment:
Inherit directly from `DQEntity` and rename to `JobState` in the same way as we would rename table.